### ISSUE TITLE
Menu item '8 Reindex repository' is default selected by mistake

### ISSUE THREAD

**Issue by EugeneManushkin** [EugeneManushkin/CtagsSourceNavigator#68]

**Comment by EugeneManushkin** [comment]
```c++
    int res=Menu(GetMsg(MPlugin),ml,MFindSymbol);//<<<<<< must be miFindSymbol instead of MFindSymbol
```
I find this behaviour convenient thow. Since no one complains I decided to not change this behaviour and also set 'Reindex repository' by default in main menu.

**Comment by EugeneManushkin** [comment]
Fixed in build 2.1.0.62
